### ISSUE TITLE
F12 stops showing qterminal dropdown

### ISSUE THREAD

**Issue by kenfallon** [lxqt/qterminal#964]
After switching desktop windows, F12 (sometimes) stops responding to showing the terminal.

It is still running
```
$ ps -ef | grep qterminal
ken       108918    3270  0 16:10 ?        00:00:23 /usr/bin/qterminal --drop
```
F12 is not mapped by the system.

Is there a way to interact with the process to get it to respond ?

What information can I provide to aid troubleshooting this issue ?

Fedora release 36 (Thirty Six) LXQT spin
  LXQt About Version:     1.0.0
  LXQt Version:           1.0.0
  Qt:                     5.15.3
  Build type:             Release
  System Configuration:   /etc/xdg
  Share Directory:        /usr/share/lxqt

**Comment by stefonarch** [comment]
Probably related to the WM. Which one are you using? Switching by keyboard shortcuts?

**Comment by tsujan** [comment]
The reason should be 1.0.0. Many problems of drop-down mode were fixed in 1.1.0, after https://github.com/lxqt/qterminal/commit/004f1ddb8ca828399aa6f9b56b608199f9dddc8b.

I think this can be closed.

**Comment by yan12125** [comment]
Yeah, sounds the same issue as https://github.com/lxqt/qterminal/issues/491, which is fixed in 1.1.0.

**Comment by yan12125** [comment]
Closing as no response after some time. Ping me or any other LXQt member if you need to reopen this issue.
